src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -151,7 +143,7 @@ const formConfig = {
           }
         },
         evidenceType: {
-          title: (formData, { pagePerItemIndex }) => _.get(`disabilities.${pagePerItemIndex}.diagnosticText`, formData),
+          title: (formData, { pagePerItemIndex }) => _.get(`disabilities.${pagePerItemIndex}.name`, formData),
```

Comment:
@goldenmeanie has indicated that at least one kind of evidence is required. We could use a UI validation to check for this, i think.

Reply:
Seems like a good idea, assuming we can confirm this is the case. I'd like to push that off into another PR.